deployment/src/main/java/com/couchbase/quarkus/extension/deployment/jacksonhandling/JacksonProcessor.java

```diff
@@ -0,0 +1,538 @@
+/*
```

Comment:
Oof - a hefty amount of code to C&P, and presumably keep up-to-date.  Remind me why we need these?  Are these the Quarkus Jackson substitutions, that we need to change to our shaded package names?

Reply:
Yes, in a project using the Quarkus extension for netty, the extension provides substitutions/code to make it native-compatible, bean injection through Quarkus' CDI, config items that go into `application.properties` etc... In a normal setup those substitutions would apply to any netty dep present.

But since we shade netty under our namespace, the substitutions don't target that code. 
Now that you bring it up we really just want the substitutions, and I could strip out the rest.